### PR TITLE
make deabsdeep compatible with windows

### LINKED ISSUE #2: Could deabsdeep replace backslashes with slashes on Windows ?
When running styleguidist tests on windows we get abunch of failures.
Could deabsdeep (on windows only) replace \ with / in managed path ?

### PR DESCRIPTION
**Pull request by elevatebart**
closes #2

### CODE REVIEW

File: src/index.js [hunk 1]
```diff
@@ -22,7 +23,16 @@ module.exports = function(obj, options) {
...
 
 	const regExp = new RegExp(escape(root), 'g');
-	const deabs = s => (typeof s === 'string' ? s.replace(regExp, mask) : s);
+	const pureDeabs = s => s.replace(regExp, mask);
+
+	const normalizeSeparators = s => (s ? s.replace(new RegExp('\\' + path.sep, 'g'), '/') : s);
+
+	const deabs = s => {
+		if (typeof s === 'string') {
+			return path.sep === '/' ? pureDeabs(s) : normalizeSeparators(pureDeabs(s));
```
Comment (sapegin) [inline src/index.js]: Do we really need a condition here?

Comment (elevatebart) [reply src/index.js]: probably not... will remove

File: src/__tests__/index.spec.js [hunk 2]
```diff
@@ -67,3 +67,10 @@ test('ignore non plain objects', () => {
 	});
 	expect(result).toMatchSnapshot();
 });
+
+test('load absolute path on windows too', () => {
+	const result = deabsDeep({
+		winPath: __dirname,
```
Comment (sapegin) [inline src/__tests__/index.spec.js]: How does this test test Windows specific behavior if we run tests on unix?

Comment (elevatebart) [reply src/__tests__/index.spec.js]: it does not... I just ran it on windows. 
I will change it.

**Comment by elevatebart** [comment]
@sapegin I rolled back the update of jest so that it keeps all tests to green.
You should be ready for release.

Thank you for making testing and snapshoting easier !

**Comment by sapegin** [comment]
Weird that it fails on Node 6 only now ;-/ 

But also we can get rid of Node <8 support and make a new major release.

**Comment by sapegin** [comment]
Done: https://github.com/sapegin/deabsdeep/releases/tag/v2.0.0

I hope it would help.

**Comment by elevatebart** [comment]
It seems it did help a lot

**Pull request merged**